geoportal/geoportailv3_geoportal/static-ngeo/less/backgroundselector.less

```diff
@@ -106,24 +106,81 @@
             top: 0;
         }
 
-        .bg-selector-layer-0 { // roadmap (mvt)
+        img.bg-selector-layer-0 { // roadmap (mvt)
             background: url(../images/backgroundselector/routiere_sm.png) no-repeat white;
         }
-        .bg-selector-layer-1 { // topographic b&w
+        img.bg-selector-layer-1 { // topographic b&w
             background: url(../images/backgroundselector/topo_nb_sm.png) no-repeat white;
         }
-        .bg-selector-layer-2 { // topographic
+        img.bg-selector-layer-2 { // topographic
             background: url(../images/backgroundselector/topo_sm.png) no-repeat white;
         }
-        .bg-selector-layer-3 { // orthophoto
+        img.bg-selector-layer-3 { // orthophoto
             background: url(../images/backgroundselector/orthophoto_sm.png) no-repeat white;
         }
-        .bg-selector-layer-4 { // hybrid orthophoto+label
+        img.bg-selector-layer-4 { // hybrid orthophoto+label
             background: url(../images/backgroundselector/hybrid_sm.png) no-repeat white;
         }
-        .bg-selector-layer-5 { // blank map
+        img.bg-selector-layer-5 { // blank map
             background-color: white;
             border: black solid 0.5px;
         }
     }
 }
+
+@media
+only screen and (-webkit-min-device-pixel-ratio: 1.25),
+only screen and ( min--moz-device-pixel-ratio: 1.25),
+only screen and ( -o-min-device-pixel-ratio: 1.25/1),
+only screen and ( min-device-pixel-ratio: 1.25),
+only screen and ( min-resolution: 200dpi),
+only screen and ( min-resolution: 1.25dppx)
+{
+    .background-layer {
+        // Retina for thumbnails
+        img.bg-selector-layer-0 { // roadmap (mvt)
+            background: url(../images/backgroundselector/routiere_retina.png) no-repeat white;
+            background-size: 90px 58px;
```

Comment:
you may set background-color, background-repeat & background size globally to avoid repeating it.
Then only set background-url per item.